core/rfb.js

```diff
@@ -2863,10 +2879,36 @@ export default class RFB extends EventTargetMixin {
                                              this._fbWidth, this._fbHeight);
     }
 
+    _updateBrowserWindows(width, height) {
+        if (this._resizeBrowser &&
+            this._rfbConnectionState === 'connected') {
+            const fbWidth  = width;
+            const fbHeight = height;
+            let OldResolutionEqual = false;
+
+            if (bodyWidthBrowserResize === document.body.clientWidth &&
+                bodyHeightBrowserResize === document.body.clientHeight) {
+                OldResolutionEqual = true;
+            }
```

Comment:
I'm not sure if this gives the desired behaviour. What we want to compare with is `_fbWidth` and `_fbHeight` just before the resize, and see if they match the browser window.

This code might still be doing that, but it's not obvious that it does.